tests/pair-count-tests.scm

```diff
@@ -0,0 +1,203 @@
+; Perform unit tests for the pair-counting part of the ULL pipeline
+
+(use-modules (opencog test-runner))
+
+(opencog-test-runner)
+; Name of test-suite
+(define suite-name "pair-count-tests")
+
+;------------------------------------------------------------------------------
+; Setup
+
+(use-modules
+  (opencog)
+  (opencog nlp)
+  (opencog persist)
+  (opencog persist-sql)
+  (opencog cogserver))
+
+; NOTE
+; 1. The files are loaded in pipeline order. In general, the later files
+;  depend on definitions contained
+; in the earlier files.
+; 2. Poc changes are loaded after files of the same name are loaded so as to
+; redfefine the functions.
+; 3. load-from-path is used so as to be able to redfine some functions. If
+; (opencog nlp learn) that will not be possilbe as some of the functions
+; are not exported.
+(load-from-path "opencog/nlp/learn/common.scm")
```

Comment:
Why don't you just do `(opencog nlp learn)` ?

Reply:
When a guile module is added using `ADD_GUILE_MODULE` then running `make` will result in the copying of the scheme files to `build_directory/pencog/scm`. So adding a test using `ADD_GUILE_TEST` will add this path when running the test.

Reply:
Are you suggesting to remove line 28 and all similar ones below? I added (opencog nlp learn) and removed these lines and then the variables in those files (i.e. inside common.scm) are not recognized.